src/lcm/function_representation.py

```diff
@@ -129,7 +131,7 @@ def get_value_function_representation(
 
 def _get_label_translator(
     in_name: str,
-) -> Callable[..., Array]:
+) -> Callable[..., Scalar]:
```

Comment:
```suggestion
) -> Callable[..., Int]:
```
? 

Same for the remainder here, some of that would be `Float`.

If we need the inclusion of 0-d Jax Arrays, we could define `IntScalar` and `FloatScalar`.